### ISSUE TITLE
Not working? in chrome

### ISSUE THREAD

**Issue by FlamelNi** [Riib11/SuperSoccer#1]
Several errors are made here with controllers and animations.

I did few testings and figured out that:
This does not work in chrome.
According to my research, javascript cannot call on other javascript file unless some other tools are used. Only exception is the latest MS Edge, which has a default function that allows javascript file connections.
So, I tried this in MS Edge, and it worked. No errors were shown.

However, in MS Edge does not allow my ps2 controllers. (and possibly my xboxOne controller).

I know this project is old and nobody really cares, I thought it would be cool if I could show this to my phaser.io presentation for club and will be working on this.

Basically I would just merge several files in to one, so that this error does not occur. What I can do at least is to put "classes" (functions/vars that contains bunch of vars) to group them and stand out.

**Comment by FlamelNi** [comment]
After few more testings, here is what I found out:
I don't know what method you used to link different javascript files, but it DOES work on chrome. The reason why it did not work for me was because I did not have the latest version of chrome. (My chrome was not so outdated. It just wasn't most recent version.) My errors disappeared as soon as I updated my chrome. So if you or others experience similar experience, just remember to update chrome in most recent version.
